### DISCUSSION TITLE
React native zero to mastery

### DISCUSSION THREAD

**Discussion by cutbypham** [OngDev/community#36] [Udemy Request]
#30 
Master React Native for iOS and Android Mobile App Development using JavaScript. Build a modern e-commerce mobile app!
https://www.udemy.com/course/complete-react-native-mobile-development-zero-to-mastery-with-hooks/

**Comment by kunal-j10** [comment]
@thuanpham2311  any coupon code available?

**Reply by cutbypham** [reply]
Not have yet
